pkg/authenticate/authenticate.go

```diff
@@ -109,3 +109,47 @@ func CreateClientFromS3Bucket(bucket, s3FileName, region, clusterCtx string) (cl
 
 	return
 }
+
+// switchContextFromConfigFile takes a kubeconfig file and a cluster context i.e. live-1/manager/live
+// and set current context, which is usefull to switch context to multiple clusters.
+func SwitchContextFromConfigFile(clusterCtx, kubeconfigPath string) error {
+	kubeconfig := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
+		&clientcmd.ClientConfigLoadingRules{ExplicitPath: kubeconfigPath},
+		&clientcmd.ConfigOverrides{
+			CurrentContext: clusterCtx,
+		})
+	config, err := kubeconfig.RawConfig()
+	if err != nil {
+		return fmt.Errorf("error getting RawConfig: %w", err)
+	}
+
+	if config.Contexts[clusterCtx] == nil {
+		return fmt.Errorf("context %s doesn't exists", clusterCtx)
+	}
+
+	config.CurrentContext = clusterCtx
+	err = clientcmd.ModifyConfig(clientcmd.NewDefaultPathOptions(), config, true)
+	if err != nil {
+		return fmt.Errorf("error ModifyConfig: %w", err)
+	}
+
+	fmt.Printf("Switched to context \"%s\"", clusterCtx)
```

Comment:
I'm not a big fan of print statements like this in public functions. Just a preference.

In your function here, it should just error if there's a problem. Let the caller decide if they want to output anything.